test/dynamo/test_higher_order_ops.py

```diff
@@ -120,7 +120,7 @@ def _assert_wrap_fallback(self, func, args, setup=lambda: None):
 
         self.assertEqual(result, expected)
 
-    def _test_wrap_simple(self, func, args, expected_num_wrap_args, expected_opcount=1):
+    def _test_wrap_simple(self, func, args, expected_num_wrap_args, expected_opcount=2):
```

Comment:
Could we add a self.assertExpectedInline test on the graph to confirm that our outputs have been flattened? One way to do this is to change `_test_wrap_simple` to return backend.graphs and then self.assertExpectedInline on the return in one of our tests

Reply:
Maybe wait for https://github.com/pytorch/pytorch/pull/109962 which does this already